<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DescargoBienes</class>
 <widget class="QWidget" name="DescargoBienes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>699</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>897974</width>
    <height>989797</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Descargo de Bienes del Inventario de Activo Fijo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Bien:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QWidget" name="widget_10" native="true">
           <layout class="QFormLayout" name="formLayout_4">
            <item row="2" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Modelo:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Marca:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="marca">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="modelo">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QWidget" name="widget_11" native="true">
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Valor: ($)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Serie:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="valorBien">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="locale">
               <locale language="Spanish" country="ElSalvador"/>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="specialValueText">
               <string/>
              </property>
              <property name="keyboardTracking">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>10000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="serie">
              <property name="maxLength">
               <number>25</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QWidget" name="widget_12" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Adquirido:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="fechaAquisicion">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QWidget" name="widget_13" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Descripcion:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="descripcion">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget_14" native="true">
           <layout class="QFormLayout" name="formLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Cuenta: </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="cuentaAsignada">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Especifico: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="nombreEspecifico">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_15" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QTableWidget" name="tablaBienes"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_16" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="botonAgregarBien">
           <property name="toolTip">
            <string>Agregar Bien</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ContablesPrincipal.qrc">
             <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="botonEliminarBien">
           <property name="toolTip">
            <string>Eliminar Bien</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../ContablesPrincipal.qrc">
             <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="nuevoBien">
           <property name="toolTip">
            <string>Nuevo Bien</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../ContablesPrincipal.qrc">
             <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_18" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Total:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="totalIngreso">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QFormLayout" name="formLayout_7">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Motivo del descargo:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="motivo"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Observaciones:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="observaciones">
        <property name="maxLength">
         <number>148</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="widget_17" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AF-M3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="guardarTraslado">
           <property name="text">
            <string>Guardar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="finalizarTransaccion">
           <property name="text">
            <string>Descargar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelar">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Codigo del descargo y bien:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cuenta:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Clase:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Correlativo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Especifico:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="cuentaNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="claseNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="correlativoNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="especificoNO">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QToolButton" name="consultaBien">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Codigo Descargo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="codigoDescargo"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Inventario:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="anioInventario"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cuentaNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>claseNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>claseNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>correlativoNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>correlativoNO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>especificoNO</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
